official/utils/misc/distribution_utils.py

```diff
@@ -111,7 +110,8 @@ def get_distribution_strategy(distribution_strategy="default",
               all_reduce_alg))
     return tf.distribute.MirroredStrategy(
         devices=devices,
-        cross_device_ops=_MIRRORED_ALL_REDUCE_OPTIONS[all_reduce_alg])
+        cross_device_ops=_MIRRORED_ALL_REDUCE_OPTIONS[all_reduce_alg](
+            num_packs=num_packs if num_packs is not None else 1))
```

Comment:
would it make sense to just make the default for the flag as 1 instead of None?

Reply:
Done.